src/authentication/helpers/redirect-to-idp.ts

```diff
@@ -1,8 +1,7 @@
 import queryString from 'query-string';
 
 import { getEnv } from '../../shared/helpers';
-import toastService, { TOAST_TYPE } from '../../shared/services/toast-service';
-import { getLogoutPath } from './get-profile-info';
+import { SERVER_LOGOUT_PAGE } from '../authentication.const';
```

Comment:
Newline boven deze import want zit in eenzelfde module met huidige file.

Reply:
Ik dacht dat wel enkel opsplitsen volgens:

* external libs
* avo libs
* files in the same project

Reply:
die laatste wordt idd nog opgesplits in:
- files buiten de module
- files in dezelfde module

Reply:
ok